doc/shell/shell_guide.md

```diff
@@ -0,0 +1,817 @@
+testcase_exclude_from_file=${HOME}/cubrid-testcases-private-ex/shell/config/daily_regression_test_excluded_list_linux.conf
+testcase_update_yn=true
+testcase_git_branch=develop
+testcase_timeout_in_secs=604800
+test_platform=linux
+test_category=shell
+testcase_exclude_by_macro=LINUX_NOT_SUPPORTED
+testcase_retry_num=0
+delete_testcase_after_each_execution_yn=false
+enable_check_disk_space_yn=true
+
+feedback_type=database
+feedback_notice_qahome_url=http://192.168.1.86:8080/qaresult/shellImportAction.nhn?main_id=<MAINID>
+
+owner_email=Mandy<cui.man@navercorp.com>
+
+git_user=cubridqa
+git_email=dl_cubridqa_bj_internal@navercorp.com
+git_pwd=N6P0Sm5U7h
+
+feedback_db_host=192.168.1.86
+feedback_db_port=33080
+feedback_db_name=qaresu
+feedback_db_user=dba
+feedback_db_pwd=
+```
+shell_template.conf will be copied to \~/CTP/conf/shell_runtime.conf when test is started.  
+For more details about the parameters, please refer to CTP guide.  
+
+### Set ~/.bash_profile 
+*~/.bash_profile*  
+```
+# .bash_profile
+
+# Get the aliases and functions
+if [ -f ~/.bashrc ]; then
+        . ~/.bashrc
+fi
+
+# User specific environment and startup programs
+
+PATH=$JAVA_HOME/bin:$HOME/CTP/common/script:$PATH:$HOME/.local/bin:$HOME/bin
+
+export PATH
+
+export CTP_BRANCH_NAME="develop"
+export CTP_SKIP_UPDATE=0
+```
+
+### Create a script to start consumer
+~/start_test.sh
+```
+nohup start_consumer.sh -q QUEUE_CUBRID_QA_SHELL_LINUX -exec run_shell &
+```
+Execute the script to start listening the test message after deployment. This will start a shell test when the consumer receive the test message.
+```
+cd ~
+sh start_test.sh
+```
+
+## 3.5 Deploy worker node  
+### Install CTP
+Please refer to ["CTP Installation Guide"](https://github.com/CUBRID/cubrid-testtools/blob/develop/doc/ctp_install_guide.md)
+
+### Set ~/.bash_profile
+*~/.bash_profile*
+```
+# .bash_profile
+
+# Get the aliases and functions
+if [ -f ~/.bashrc ]; then
+        . ~/.bashrc
+fi
+
+# User specific environment and startup programs
+
+PATH=$PATH:$HOME/.local/bin:$HOME/bin
+
+export CTP_HOME=$HOME/CTP
+## init_path is used when we run shell case manually on this machine.
+export init_path=$CTP_HOME/shell/init_path
+
+export PATH=$CTP_HOME/bin:$CTP_HOME/common/script:$PATH
+
+export CTP_BRANCH_NAME="develop"
+export CTP_SKIP_UPDATE=0
+
+. ~/.cubrid.sh
+export GCOV_PREFIX=/home/shell
+export GCOV_PREFIX_STRIP=2
+ulimit -c unlimited
+```
+
+### Deploy test cases
+```
+git clone --no-checkout https://github.com/CUBRID/cubrid-testcases-private-ex.git
+cd ~/cubrid-testcases-private-ex
+git config core.sparseCheckout true
+echo 'shell/*' > ~/cubrid-testcases-private-ex/.git/info/sparse-checkout
+git checkout develop
+```
+
+### Make directories for test
+```
+cd
+mkdir do_not_delete_core
+mkdir ERROR_BACKUP
+```
+
+### Create .cubrid.sh file 
+If cubrid has never been installed on the machine, we need create file '.cubrid.sh' at $HOME path manually.  

```

Comment:
It's enough to add ". .cubrid.sh" to .bash_profile.
Needn's copy full content of it.